dist/routable-component.js

```diff
@@ -381,37 +381,37 @@ var routableComponent =
 	var RoutableComponentRoutes = function () {
 	  /**
 	   * コンストラクタ
-	   * @param {RoutableComponentRouting|RoutableComponentRouting[]} routing_classes ルート定義クラス(の配列)
+	   * @param {RoutableComponentRouting|RoutableComponentRouting[]} routingClasses ルート定義クラス(の配列)
 	   */
 	
 	  function RoutableComponentRoutes() {
-	    var routing_classes = arguments.length <= 0 || arguments[0] === undefined ? [] : arguments[0];
+	    var routingClasses = arguments.length <= 0 || arguments[0] === undefined ? [] : arguments[0];
 	    (0, _classCallCheck3.default)(this, RoutableComponentRoutes);
 	
 	    this._routes = [];
-	    this.include_route(routing_classes);
+	    this.includeRoute(routingClasses);
 	  }
 	
 	  /**
 	   * ルートを設定する
-	   * @param {Route|Route[]} routing_classes ルート定義クラス(の配列)
+	   * @param {Route|Route[]} routingClasses ルート定義クラス(の配列)
 	   * @return {void}
 	   */
 	
 	
 	  (0, _createClass3.default)(RoutableComponentRoutes, [{
-	    key: 'include_route',
-	    value: function include_route(routing_classes) {
-	      var _routing_classes = routing_classes instanceof Array ? routing_classes : [routing_classes];
+	    key: 'includeRoute',
+	    value: function includeRoute(routingClasses) {
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)
* Warning - Function 'includeRoute' has a complexity of 6. (complexity)
* Warning - This function has too many statements (16). Maximum allowed is 10. (max-statements)

<!-- Comment by [SideCI](https://sideci.com) -->